### PR TITLE
Display drawer claudy section only if claudy actions in context

### PR DESCRIPTION
**Pull request by CPatchane**

### CODE REVIEW

File: src/components/Bar.svelte [hunk 1]
```diff
@@ -72,6 +72,12 @@
         await updateSettings(config)
         await updateApps(config)
       }
+      // filter claudy subsection if claudyConfig is null
+      if (!claudyConfig) {
+        if (config.sections && config.sections.drawer) {
+          config.sections.drawer = config.sections.drawer.filter(s => s.length && s[0].slug !== 'claudy')
```
Comment (gregorylegarec) [inline src/components/Bar.svelte]: What means `s` ?

Comment (CPatchane) [inline src/components/Bar.svelte]: Subsections, I used just the initial letter in functions like `sort()` or `filter()` to keep them short if they can be written in one line.

**Comment by gregorylegarec** [comment]
Thanks @CPatchane !

File: src/components/Bar.svelte (hunk 1 above)
Comment (gregorylegarec) [inline src/components/Bar.svelte]: Ok, I am not a big fan of _one-letter variables_, but let's keep this one. 😄

**Pull request merged**